poetry.lock

```diff
@@ -1,10 +1,9 @@
-# This file is automatically @generated by Poetry 1.4.0 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
 
 [[package]]
 name = "et-xmlfile"
 version = "1.1.0"
 description = "An implementation of lxml.xmlfile for the standard library"
-category = "main"
```

Comment:
@gurevichdmitry 

any idea why this line was removed multiple times ? is it because i used poetry `1.5.1` ?

Reply:
@orouz, you are correct. There has been a change since the release of Poetry [1.5.0]( https://github.com/python-poetry/poetry/releases/tag/1.5.0)